src/errno.rs

```diff
@@ -103,6 +104,12 @@ impl Display for Error {
     }
 }
 
+impl StdError for Error {
```

Comment:
Huh? None is the trait default.. No need to implement that

fn source(&self) -> Option<&(dyn Error + 'static)> { None }

Reply:
ah right it makes sense.....didn't thought of that as I only checked the [trait defination](https://doc.rust-lang.org/std/error/trait.Error.html#method.source) in the offical doc. Thanks for pointing it out. 

I already saw your PR to fix it, thanks for that too :)